src/config/templates.toml

```diff
@@ -20,11 +20,11 @@ label(if(current_working_copy, "working_copy"), concat(
     git_head,
     format_short_commit_id(commit_id),
     if(conflict, label("conflict", "conflict")),
-  ) "\n",
+  ) ++ "\n",
```

Comment:
Why not use a comma instead of ++ here? (Also, below and in the op log template)

Reply:
Just for consistency. Each `concat()` argument is a single line.